### ISSUE TITLE
Working sample using the plugin

### ISSUE THREAD

**Issue by gouthamkumar253** [jonasbark/flutter_stripe_payment#92]
Hi Jonas, I have written an article addressing a few common doubts about this package and published it in medium. Check it out when you find time.

https://medium.com/flutter-community/flutter-card-payments-made-easy-with-stripe-and-ruby-on-rails-50d78ac056ce

**Comment by assad62** [comment]
this is awesome. Thanks !

**Comment by Pebsie** [comment]
This article is almost exactly what I was looking for, but I can't seem to see any explanation for it switching to Ruby halfway through. What's the deal with that? What does that have to do with Flutter?

**Comment by gouthamkumar253** [comment]
Hi @Pebsie , the reason for including the ruby part is that since in most cases you might need to save the details in a database, I thought I'd cover that too... Also, the initial plan was to make the article a two-part series where I thought of explaining how to make Stripe Connect accounts and manage the escrow payments. For that, I needed a lot of backend functionalities so thought of giving an intro to rails in the first part itself. Sorry if that was unnecessary or confusing.

**Comment by Pebsie** [comment]
Not at all! Sorry for my tone, I was getting flustered after many hours searching. What wasn't obvious to me at the time is that payments need to go through a backend in the first place -- a word on this in the article would be really useful for others in a similar position! :)

**Comment by jonasbark** [comment]
Please note it does not *have to*. You could still make the necessary payment requests to Stripes' API within Dart - but generally it's wise to do it on a server to not accidently expose important secrets within the App.
